### PR TITLE
Add EIP: Multi-User NFT Extension

### PR DESCRIPTION
**Pull request by minkyn**

### CODE REVIEW

File: EIPS/eip-nft_multi_user.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
```
Comment (abcoathup) [inline EIPS/eip-nft_multi_user.md:1]:
```suggestion
---
eip: 7507
```
Assigning next sequential EIP/ERC number.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

**Review by Aryan9592 — COMMENTED** [review]
Nice Attempt

File: EIPS/eip-nft_multi_user.md (hunk 1 above)
Comment (Mikh333) [reply EIPS/eip-nft_multi_user.md]: {
    "id": 1337
    "jsonrpc": "2.0",
    "error": {
        "code": -32003,
        "message": "Transaction rejected"
    }
}

**Comment by aa-eyup** [comment]
You might find [this project](https://github.com/aa-eyup/access-contracts-mo) interesting - it tries to provide similar functionality. It Some major differences is that 1) ability to access a given NFT is done through ownership of a token representing that specific access type 2) existing NFTs can "plug-in", inheriting functionality is unnecessary 3) revenues generated from selling access to a given NFT can be distributed to multiple accounts based on ownership share of the NFT

**Comment by minkyn** [comment]
>

File: EIPS/eip-7507.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+However, we didn't include the corresponding `usersOf(uint256 tokenId)` function as that would imply the implemention has to support enumerability over multiple users. It is not always necessary, for example, in the case of open subscription. So we decide not to add it to the interface and leave the choice up to the implementers.
+
+## Backwards Compatibility
+
+No backwards compatibility issues found.
+
+## Test Cases
+
+Test cases available available at: [ERC7507.test.ts](../assets/eip-7507/test/ERC7507.test.ts):
```
Comment (xinbenlv) [inline EIPS/eip-7507.md:61]:
```suggestion
Test cases available available at: [`ERC7507.test.ts`](../assets/eip-7507/test/ERC7507.test.ts):
```

**Pull request merged**